mkosi

```diff
@@ -558,6 +558,9 @@ def attach_image_loopback(args: CommandLineArguments, raw: Optional[IO[str]]):
             run(["losetup", "--detach", loopdev], check=True)
 
 def partition(loopdev: str, partno: int) -> str:
```

Comment:
So please fix the type annotation too. It should be `Optional[int]` in this case.

Reply:
The annotations are not complete, and there's hundreds of warnings reported by mypy, so it's pretty hard to notice new bugs. The longer term goal is to clean this all up, but we're not there yet.